mllm/backends/qnn/aot/passes/PTQPass.cpp

```
@@ -0,0 +1,230 @@
+    if (op->isa_<ir::linalg::LinearOp>()) { solveLinearWeight(w.getContext(), pf, op->cast_<ir::linalg::LinalgIROp>()); }
+    if (op->isa_<ir::linalg::RMSNormOp>()) { solveRMSNormWeight(w.getContext(), pf, op->cast_<ir::linalg::LinalgIROp>()); }
+    if (op->isa_<ir::linalg::EmbeddingOp>()) { solveEmbeddingWeight(w.getContext(), pf, op->cast_<ir::linalg::LinalgIROp>()); }
+    if (op->isa_<ir::graph::CallGraphOp>()) {
+      auto ns = op->cast_<ir::graph::CallGraphOp>()->getSymbolAttr()->str();
+      recursiveSolveWeights(w.getContext(), w.getContext()->lookupSymbolTable(ns)->cast_<ir::graph::SubGraphOp>(), pf);
+    }
+    return ir::IRWriter::WALK_CONTINUE;
+  });
+}
+
+void _recursiveSolveNormalImpl(const ir::Val::ptr_t& v) {
```

Clang (14.0.6)

[error] parameter name 'v' is too short, expected at least 3 characters

(readability-identifier-length,-warnings-as-errors)